VAMobile/src/screens/PaymentsScreen/DirectDepositScreen/DirectDepositScreen.tsx

```diff
@@ -40,9 +41,15 @@ function DirectDepositScreen({ navigation }: DirectDepositScreenProps) {
     isLoading: loading,
     isError: useBankDataError,
   } = useBankData({ enabled: screenContentAllowed('WG_DirectDeposit') && ddNotInDowntime })
-  const bankData = directDepositData?.data.attributes?.paymentAccount
+  const [bankData, setBankData] = useState(directDepositData?.data.attributes?.paymentAccount)
   const { gutter, contentMarginBottom } = theme.dimensions
 
+  useFocusEffect(
```

Comment:
Is the useFocusEffect needed because the component doesn't re-render when directDepositData changes?

Reply:
that is correct, when we manually alter the data with setQuery it doesn't force a re render because we navigate backwards. If we left the screen and re entered it would be fine.